lib/blockscore.rb

```diff
@@ -1,5 +1,15 @@
-require "httparty"
+require 'blockscore/base'
+require 'blockscore/candidate'
+require 'blockscore/company'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.